react/features/app/components/AbstractApp.js

```diff
@@ -39,6 +42,14 @@ export class AbstractApp extends BaseApp<Props, *> {
      * @inheritdoc
      */
     componentDidMount() {
+        if (!isInIframe() && config.hosts.parent) {
+            const baseUrl = window.location.href;
+            const parent = config.hosts.parent;
+            const queryStringAndFragment = getQueryStringAndFragment(baseUrl);
+            const redirectTo = queryStringAndFragment ? `${parent}${queryStringAndFragment}` : parent;
+
+            window.location = redirectTo;
+        }
```

Comment:
```suggestion
                return;
        }
```

Reply:
this will stop the things that execute afterwards, since window.location takes some time to execute and script continues with the rest of the code